BackEnd/Medical System/Controllers/ReservationController.cs

```diff
@@ -73,6 +73,20 @@ public async Task<IActionResult> GetTodayReservations([FromQuery]PageFilter filt
             return this.SuccessCollection(response);
         }
 
+        [HttpGet("UserReservations")]
+        public async Task<IActionResult> GetUserReservationsAsync([FromQuery] string userId)
+        {
+            var response = await _service.GetUserReservationsAsync(userId);
+
+            if (response.Succeeded)
+            {
+                return this.CreateResponse(response);
+            }
+            else
+            {
+                return BadRequest(response.Message);
```

Comment:
same as previos one bro
delete the whole condition and just return this.creatersponse(response);